### PR TITLE
feat: Enable interrupting conversation

### PR DESCRIPTION
**Pull request by hofmeister**

### CODE REVIEW

File: src/renderer/components/storm/context/StormChatContext.tsx [hunk 1]
```diff
@@ -26,15 +28,21 @@ const StormChatContext = React.createContext<StormChatContextType | undefined>(u
 interface StormChatContextProviderProps {
     children: ReactNode;
     conversation?: AIConversationThread;
+    loading?: boolean;
```
Comment (andertun) [inline src/renderer/components/storm/context/StormChatContext.tsx:31]: Maybe a bit confusing to have `loading` as prop when we already have it in state?

File: stories/henrik.stories.tsx [hunk 2]
```diff
@@ -85,11 +86,14 @@ export const StreamingView = () => {
...
         setIndex(index + 1);
+        if (index >= events.length) {
+            //setLoading(false);
+        }
     }, 400);
 
     return (
         <DefaultContext>
-            <StormPlanView conversation={conversation} />
+            <StormPlanView conversation={conversation} loading={loading} />
```
Comment (andertun) [inline stories/henrik.stories.tsx:96]: Will `loading` be used?

**Review by andertun — APPROVED** [review]

File: src/renderer/components/storm/context/StormChatContext.tsx (hunk 1 above)
Comment (hofmeister) [reply src/renderer/components/storm/context/StormChatContext.tsx:39]: Just to be able to provide it from storybook

File: stories/henrik.stories.tsx (hunk 2 above)
Comment (hofmeister) [reply stories/henrik.stories.tsx:96]: Only in storybook

**Pull request merged**